docs/software-platforms/aws/starting-development-on-lsfr.md

```diff
@@ -0,0 +1,78 @@
+# Getting started with development on Lightsail for Research
+
+Once you have [created a virtual computer](creating-vcs.md), [attached a disk](working-with-volumes.md), and [accessed the operating system](creating-vcs.md#how-to-access-a-virtual-computer) you are ready to start developing analyses on Lightsail for Research.
+
+## Quick tips about virtual computers
```

Comment:
I think this section is good. If we start running into common issues then we can always expand it later, but I can't think of anything else that might be necessary for people to know at this point.